sale_procurement_group_by_line/model/sale.py

```diff
@@ -20,6 +20,14 @@ def _prepare_procurement_group_by_line(self, line):
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
+    def _get_procurement_group(self):
+        """Standard Odoo resturns the group of the SO but here we rely on the
+        group of the line"""
```

Comment:
Nitpicking: this should be a comment, not a docstring

Reply:
done! :)